### PR TITLE
UK Network front heatmap

### PR DESCRIPTION
**Pull request by kenlim**
Small AB test to generate a heatmap of the UK network front.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/heatmap.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        this.description = 'UK network front heatmap';
+        this.audience = 0.01;
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = '1% looking at the UK front';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            return (config.page.edition === 'UK') && config.page.pageId === "uk"  ;
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/experiments/tests/heatmap.js]: why there is so much space before ';' ?

File: static/src/javascripts/projects/common/modules/experiments/tests/heatmap.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+        this.id = 'Heatmap';
+        this.start = '2015-03-16';
+        this.expiry = '2015-05-18';
```
Comment (phamann) [inline static/src/javascripts/projects/common/modules/experiments/tests/heatmap.js]: This date does not map to your switch expiration date?

**Comment by phamann** [comment]
:+1: Other than comment

**Comment by austinh7** [comment]
I think we need to run this on an article page as well. (And possibly a lower impact front?)

**Pull request merged**